src/timeseriesinsights/azext_timeseriesinsights/custom.py

```diff
@@ -0,0 +1,225 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
+
+
+def create_timeseriesinsights_environment_standard(cmd, client,
+                                                   resource_group_name, environment_name,
+                                                   sku_name, sku_capacity,
+                                                   data_retention_time,
+                                                   storage_limit_exceeded_behavior=None,
+                                                   partition_key_properties=None,
+                                                   location=None, tags=None):
+    from .vendored_sdks.timeseriesinsights.models import StandardEnvironmentCreateOrUpdateParameters, Sku, \
+        TimeSeriesIdProperty
+
+    parameters = StandardEnvironmentCreateOrUpdateParameters(
+        location=location,
+        tags=tags,
+        sku=Sku(name=sku_name, capacity=sku_capacity),
+        data_retention_time=data_retention_time,
+        storage_limit_exceeded_behavior=storage_limit_exceeded_behavior,
+        # Need to confirm whether multiple key properties are supported
+        partition_key_properties=[TimeSeriesIdProperty(name=id_property, type="String") for id_property in partition_key_properties]
+    )
+    return client.create_or_update(resource_group_name=resource_group_name, environment_name=environment_name, parameters=parameters)
```

Comment:
For commands supporting no wait, you need wrap the method with `sdk_no_wait`, for example, `sdk_no_wait(no_wait, client.create_or_update, resource_group_name, cluster_name, cluster_instance)`

Reply:
✔